Cartfile.private

```diff
@@ -1,3 +1,3 @@
 github "typelift/SwiftCheck"
-github "typelift/Operadics"
+github "typelift/Operadics" "fix-directory-structure"
```

Comment:
I'll remove this once https://github.com/typelift/Operadics/pull/10 lands (and before landing this PR). I put this here temporarily to try to get CI to pass